Formula/apib.rb

```diff
@@ -14,28 +14,12 @@ class Apib < Formula
     sha256 "bbe9bc25a8584f163347662675d78b69cdfaac495be5f2fa026dfca112f8d4a4" => :sierra
   end
 
-  depends_on "apr"
-  depends_on "apr-util"
-  depends_on "openssl@1.1"
```

Comment:
Does it not require openssl anymore?

Reply:
No -- during the Bazel build it fetches BoringSSL from the repo and builds it. That makes maintenance across platforms a lot easier for me, but if it's not a best practice for Homebrew please let me know and I can see how else I could do it.

Reply:
Yeah, pulling in dependencies over the internet is a big no-no for homebrew.https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things